.vscode/snippets.code-snippets

```diff
@@ -0,0 +1,26 @@
+{
+	// Place your azure-sdk-for-c workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and 
+	// description. Add comma separated ids of the languages where the snippet is applicable in the scope field. If scope 
+	// is left empty or omitted, the snippet gets applied to all languages. The prefix is what is 
+	// used to trigger the snippet and the body will be expanded and inserted. Possible variables are: 
+	// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. 
+	// Placeholders with the same ids are connected.
+	// Example:
+	// "Print to console": {
+	// 	"scope": "javascript,typescript",
```

Comment:
can we change the example to C99

Reply:
I just removed the example, it was autogenerated by vscode. Could also remove the snippet but imo it's handy